.ci/Jenkinsfile

```diff
@@ -15,6 +15,8 @@
 // specific language governing permissions and limitations
 // under the License.
 
+// https://apm-ci.elastic.co/job/apm-shared/job/ecs-logging-update-specs-mbp/
```

Comment:
If you prefer we can use the build badge in the README.md, if so

You can add:

```
[![Build Status](https://apm-ci.elastic.co/job/apm-shared/job/ecs-logging-update-specs-mbp/job/master/badge/icon)](https://apm-ci.elastic.co/job/apm-shared/job/ecs-logging-update-specs-mbp/job/master/)
```

For instance [![Build Status](https://apm-ci.elastic.co/job/apm-shared/job/ecs-logging-update-specs-mbp/job/master/badge/icon)](https://apm-ci.elastic.co/job/apm-shared/job/ecs-logging-update-specs-mbp/job/master/)

Reply:
I think maybe not a badge for this repo that says "build" because the Jenkinsfile in this case isn't doing anything for *this* repo and it would be slightly misleading.